documentation/modules/cruise-control/con-cruise-control-overview.adoc

```diff
@@ -28,16 +29,18 @@ They are categorized as follows:
 
 * *Hard goals* are preset and mandatory for a proposal to succeed.
 * *Soft goals* are optional and can be set aside if hard goals are met.
-* *Main goals* are inherited from Cruise Control, some preset as hard goals, used by default.
-* *Default goals* are the same as main goals by default but customizable.
+* *Main goals* are inherited from Cruise Control and typically used as the default set of goals, with some preset as hard goals.
+The goals can be customized to specify a subset of all available goals provided by Cruise Control.
```

Comment:
What about:

```
* *Main goals* contains the list of goals that can be used by the Cruise Control operations (all built-in goals by default).
This list can be customized to specify a subset of all the available goals, which may include custom goals.
```

My intention is to avoid giving the impression that you should update this configuration to change default goals. You can, but it would be an indirect update, and your configuration may be lost if someones arrives and sets `default.goals`.